worker/src/main.rs

```diff
@@ -1,3 +1,9 @@
-fn main() {
-    println!("Hello, world!");
-}
+use sysinfo::System;
+mod metrics;
+use crate::metrics::{Metric, MemoryMeasure};

```

Comment:
```suggestion

use crate::metrics::{Metric, MemoryMeasure};

mod metrics;
```

Sorry, more nitpicks. I'll configure rustmft later so that this is enforced automatically by the formatter.